### ISSUE TITLE
AssemblyBuilder documentation for .NET Core/.NET 5 is stale

### ISSUE THREAD

**Issue by vitek-karas** [dotnet/dotnet-api-docs#5321]
Some of the information on the `AssemblyBuilder` class page: https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Reflection.Emit/AssemblyBuilder.xml is describing the behavior of this API on .NET Framework not on .NET Core/.NET 5.

For example:
* All mentions of "saving the assembly" should be removed, the save functionality is not available in .NET Core
* > To get an AssemblyBuilder object, use the AppDomain.DefineDynamicAssembly method.

    The `AppDomain.DefineDynamicAssembly` doesn't exist in .NET Core. The way to create the builder object is different now
* Basically any mention of AppDomain are stale as well
* AssemblyBuilderAccess enum has different values in .NET Core - the document doesn't reflect that change

**Comment by MSDN-WhiteKnight** [comment]
I think this is now addressed after updates in #5491 and #7619 

Also duplicate issues could be closed: #2645, #4207

**Comment by buyaa-n** [comment]
> I think this is now addressed after updates in #5491 and #7619

Thanks @MSDN-WhiteKnight looks most of it addressed with https://github.com/dotnet/dotnet-api-docs/pull/5491 but it  is not covered `The AppDomain.DefineDynamicAssembly doesn't exist in .NET Core. The way to create the builder object is different now` part, i still see it in the doc https://github.com/dotnet/dotnet-api-docs/blob/17c58dc9d579eba45f6f20bb08ad5ff0a63dec47/xml/System.Reflection.Emit/AssemblyBuilder.xml#L72

> Also duplicate issues could be closed: #2645, #4207

Thanks, closed
